templates/hawk_apiserver_json_config.j2

```diff
@@ -0,0 +1,33 @@
+{%- from "cluster/map.jinja" import cluster with context -%}
+
+{
+  "key": "/etc/hawk/hawk.key",
+  "cert": "/etc/hawk/hawk.pem",
+  "listen": "0.0.0.0",
+  "port": {{ cluster.exporter.exposition_port|default(9001) }},
```

Comment:
@ayoub-belarbi I think this port collides with default `node_exporter`, doesn't it?

Reply:
I think we should perhaps create a Plain.txt of our services..
So we have schema for the future.

Reply:
``` = ["192.168.110.19:8001", "192.168.110.20:8001", "192.168.110.19:9100", "192.168.110.20:9100"]
```

Reply:
Actually, I checked this reference before assigning that: https://github.com/prometheus/prometheus/wiki/Default-port-allocations which is very useful and probably we can add our exporters there. You can check here as well  the know used ports  https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers and we can change it if you see some risk

Reply:
I think it is fine, I just was wondering if we should have a file where we collect our services as documentation.

Reply:
Yes we should have one IMHO.